schemas/oic.r.sacl.json

```diff
@@ -6,32 +6,23 @@
     "oic.r.sacl": {
       "type": "object",
       "properties": {
-        "aclist":  {
+        "aclist2":  {
           "type": "object",
```

Comment:
This needs to be an "array" not an "object". It could be an object before since the definition of "ace" contained an array when this resource was defined.

Have a look at the "aclist2" definition in oic.r.acl2. It looks like the idea is that this definition should look just like the ACL2 definition, but with a signature, correct?